kubeadm/flannel/build.ps1

```diff
@@ -0,0 +1,43 @@
+param(
```

Comment:
Will this script work in powershell core?

Reply:
Yes, it will. I checked it on powershell core.